From 7e93160ec26d59ec6d8a35a697f64fbad54efa37 Mon Sep 17 00:00:00 2001
From: Gilbert Song <songzihao1990@gmail.com>
Date: Wed, 6 Feb 2019 16:07:45 -0800
Subject: [PATCH] Cleaned up command executor redundant command string.

(cherry picked from commit b4f3c6cc2d5d2f6e3411cbc97a2feae0ed091529)
(cherry picked from commit 9a6ae80b495cb35693fdba5f44c60cd8af9dc196)
---
 src/launcher/executor.cpp | 11 +----------
 1 file changed, 1 insertion(+), 10 deletions(-)

diff --git a/src/launcher/executor.cpp b/src/launcher/executor.cpp
index 81935eb..1984486 100644
--- a/src/launcher/executor.cpp
+++ b/src/launcher/executor.cpp
@@ -486,15 +486,6 @@ protected:
     initPath = "/proc/self/fd/" + stringify(memFd.get());
 #endif // __linux__
 
-    // TODO(tillt): Consider using a flag allowing / disallowing the
-    // log output of possibly sensitive data. See MESOS-7292.
-    string commandString = strings::format(
-        "%s %s <POSSIBLY-SENSITIVE-DATA>",
-        initPath,
-        MesosContainerizerLaunch::NAME).get();
-
-    LOG(INFO) << "Running '" << commandString << "'";
-
     // Fork the child using launcher.
     vector<string> argv(2);
     argv[0] = MESOS_CONTAINERIZER;
@@ -518,7 +509,7 @@ protected:
         {Subprocess::ChildHook::SETSID()});
 
     if (s.isError()) {
-      ABORT("Failed to launch '" + commandString + "': " + s.error());
+      ABORT("Failed to launch task subprocess: " + s.error());
     }
 
     return s->pid();
-- 
2.5.1

